kapitan/inputs/helm/template.go

```diff
@@ -301,7 +301,7 @@ func writeToFile_c(outputDir string, name string, data string) error {
 		return err
 	}
 
-	// fmt.Printf("wrote %s\n", outfileName)
```

Comment:
Remove comment or change to Logger.debug. It's ok if it prints it as Debug output. :)

Reply:
I'll remove the comment since the bug is already fixed!